Eli250/my-brand
# #hhvpVbmg Validate Forms

###What does this PR do?

Validate all forms

###Description of Task to be completed?

This PR manipulates and validates user inputs from contact, signin and dashboard pages.

###How should this be manually tested?

You can evaluate this by checking if the user can see warnings in case of wrong inputs.
You can also check if user inputs are stored inside the local storage object.

###Relevant pivotal tracker stories?
###What does this PR do?

Create signin and admin panel

###Description of Task to be completed?

This PR creates the contact and sigin page for the admin panel.

###How should this be manually tested?

Check if all text inputs are provided and validated
Entering wrong input an check if the system shows error message
Enter credential and check if you reach the admin page
###Relevant pivotal tracker stories?

hhvpVbmg

###Screenshots

![s2](https://user-images.githubusercontent.com/60324751/149533006-696b5dae-6b82-41d6-afa4-5edf8b464ec3.png)

![s5](https://user-images.githubusercontent.com/60324751/149533019-9f7c0474-320d-49dc-bb0b-437eafca6e17.png)

![s4](https://user-images.githubusercontent.com/60324751/149533014-8fb03f84-fd1e-4ea1-a916-f778b858660d.png)

![s3](https://user-images.githubusercontent.com/60324751/149533010-600919a1-3775-4444-89cf-3c56e89e966c.png)

![s1](https://user-images.githubusercontent.com/60324751/149532976-11e67e17-d203-499e-8dbd-0ac40e893810.png)
